src/App.jsx

```
@@ -973,7 +1126,12 @@ export default function App() {
 
   const filteredRuns=runs.filter(r=>(brFilter==="all"||(r.branch||"main")===brFilter)&&(!histSearch||(r.goal||"").toLowerCase().includes(histSearch.toLowerCase())));
 
-  if(!landed)return <Landing onStart={()=>setLanded(true)} onSignIn={()=>{setLanded(true);setShowAuth(true);}}/>;
+  if(!landed)return <Landing
+    onStart={()=>setLanded(true)}
+    onStartWithGoal={g=>{setGoal(g);setLanded(true);}}
+    onSignIn={()=>{setLanded(true);setShowAuth(true);}}
+    onUpgrade={plan=>{setLanded(true);setShowUpg(true);}}
```

ESLint

[error] 'plan' is defined but never used.

(no-unused-vars)